documentation/01-Prerequisites.md

```diff
@@ -7,16 +7,16 @@ These are one-time setup instructions that should be executed prior to following
 - [Prerequisites for building and using MLOS](#prerequisites-for-building-and-using-mlos)
   - [Contents](#contents)
   - [Linux](#linux)
-    - [Linux Requirements](#linux-requirements)
+    - [Linux Distribution Requirements](#linux-distribution-requirements)
     - [Clone the repository](#clone-the-repository)
-    - [Linux Docker Install](#linux-docker-install)
-    - [Install Linux Build Tools](#install-linux-build-tools)
-      - [Docker Build Image](#docker-build-image)
-      - [Manual Build Tools Install](#manual-build-tools-install)
+    - [Option 1: Linux Docker Build Env](#option-1-linux-docker-build-env)
+      - [Install Docker](#install-docker)
+      - [Build the Docker Image](#build-the-docker-image)
+    - [Option 2: Manual Build Tools Install](#option-2-manual-build-tools-install)
     - [Linux Python Install](#linux-python-install)
-      - [Docker Python Install](#docker-python-install)
-      - [Using Conda](#using-conda)
-      - [Manual Python Install](#manual-python-install)
+        - [Option 1: Docker Python Install](#option-1-docker-python-install)
```

Comment:
This probably needs some rewriting abut structuring it this way first should be helpful.